docs/documentation-generation/docs-json.md

```diff
@@ -100,22 +100,48 @@ CSS file like the following:
 
 Then you'd get the following in the JSON output:
 
-```json
+```json title="Example docs-json Output"
+[
+  {
+    "name": "--background",
+    "annotation": "prop",
+    "docs": "Background of the button"
+  },
+  {
+    "name": "--background-activated",
+    "annotation": "prop",
+    "docs": "Background of the button when activated"
+  },
+  {
+    "name": "--background-focused",
+    "annotation": "prop",
+    "docs": "Background of the button when focused"
+  }
+]
+```
+
+If the style sheet is configured to be used with [a specific mode](../components/styling.md), the mode associated with
+the CSS property will be provided as well:
+
+```diff title="Example docs-json Output with Mode"
 "styles": [
```

Comment:
```suggestion
[
```

Reply:
Bah, I had that as a pending comment, but never submitted it. Thanks!